dScripts/ai/ACT/ActPlayerDeathTrigger.cpp

```diff
@@ -1,5 +1,7 @@
 #include "ActPlayerDeathTrigger.h"
 
+#include "Entity.h"
```

Comment:
Is this needed?

Reply:
Prevents some IDE errors when viewing the file